slider.js

```diff
@@ -0,0 +1,109 @@
+const SLIDER_CLASSES = {
+  LIST: ".slider-list",
+  ITEM: ".slider-item",
+  ACTIVE_ITEM: ".slider-item active",
+  PLACEHOLDER: ".placeholder",
+  ADD_SHOW_PLACEHOLDER: "show-placeholder",
+  NAVIGATION_BUTTONS: ".navigation-buttons",
+  ADD_NO_NAVIGATION_BUTTONS: "no-navigation",
+  BTN_PREV: ".btn-prev",
+  BTN_NEXT: ".btn-next",
+};
+
+const createSlider = (function () {
+  let activeSlideIndex;
+
+  const getSlides = (sliderRoot) => {
```

Comment:
single responsibility principle is broken. getSlides should only return slides. showPlaceholder is another function. Please separate them.